android/engine/src/main/resources/strings_sw.properties

```diff
@@ -0,0 +1,2 @@
+name.title=Bwana.
+gender.male=Kiume
```

Comment:
Do you think we need these in `src.main` package of engine module? We do have the some properties in `src.test` for testing.